### DISCUSSION TITLE
How do I use !important with unitless properties

### DISCUSSION THREAD

**Discussion by divyam234** [garronej/tss-react#166] [Q&A]
For example in jss I can do like fontSize: [[30], '!important']. Is there a way to use the same in tss

**Comment by garronej** [comment]
Hello

`20px !important`

**Comment by divyam234** [comment]
That is fine but it  will  be  a string unlike in jss makeStyles  I can  use property without  units as mentioned in earlier comment. Although to overcome this I am using  emotion cache with  prepend:true  so !important is not needed at all with unitless properties.

**Comment by garronej** [comment]
When there is no explicit unit it's pixel (I think) 

But i'm glad you fixed your problem
